Cargo.toml

```diff
@@ -110,7 +110,7 @@ quote = "1"
 rand = "0.8"
 rcgen = { version = "0.9", features = ["pem", "zeroize"] }
 regex = "1.4"
-reqwest = { version = "0.11", default-features = false }
+reqwest = { version = "0.11", default-features = false, features = ["trust-dns"] }
```

Comment:
This `reqwest` crate is used only by a few crates in our repo like the `downloader` crate. The major ones like `c8y_http_proxy` use `hyper` directly. So, this might only partially solve the problems.